install.md

```diff
@@ -0,0 +1,100 @@
+
+## Pre-requisites
+
+You must have a the ability to run root containers in a cluster
+with at least 12 vCPUs and 45G of memory and a minumum of three nodes.
+You will also need to provide a docker compatable registry. 
+
+## Configure your Registry
+In order to make this install simple to walk through we recomend you 
+store your docker registry details in a an environment variable. This
+install guide uses gcr on gke. 
+
+```
+export KF_REGISTRY=gcr.io/<PROJECT_ID>
+```
+
+
+## Install Istio && Knative
+
+Install istio CRDs and deploy pods and label the default namespace. 
+```
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/istio-crds.yaml && \
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/istio.yaml && \
+kubectl label namespace default istio-injection=enabled
+```
+
+Install knative CRDs.
+```
+kubectl apply --selector knative.dev/crd-install=true \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/serving.yaml \
+--filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
+--filename https://github.com/knative/eventing/releases/download/v0.5.0/release.yaml \
+--filename https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/monitoring.yaml \ &&
+--filename https://raw.githubusercontent.com/knative/serving/v0.5.0/third_party/config/build/clusterrole.yaml 
+```
+
+Install Knative PODs
+```
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/serving.yaml \
+--filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
+--filename https://github.com/knative/eventing/releases/download/v0.5.0/release.yaml \
+--filename https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/monitoring.yaml \
+--filename https://raw.githubusercontent.com/knative/serving/v0.5.0/third_party/config/build/clusterrole.yaml
+```
+
+
+## Upload buildpacks
+Buildpacks are provided by the operator and can be uploaded to Knative using the
+CLI. A set of buidpacks is included in this repo. Change into the `buildpack-samples`
+directory run the following command. 
+
+```
+kf upload-buildpacks --container-registry $KF_REGISTRY
+```
+
+## Push your first app
```

Comment:
This makes me think we need sample apps.

Reply:
We're going to be fully cf compatible so we can share those ;-)

Reply:
I was thinking sample apps with buildpack samples for now is a good idea and eventually plan to break them out.

Reply:
@josephlewis42 Do you have a link in mind?